api/src/main/resources/client_api.yaml

```diff
@@ -207,8 +207,15 @@ definitions:
       filters:
         description: >
           Filters on the results. Only results matching the criteria specified in the filters
-          will be returned
+          will be returned. If both filters and conceptSetName are specified, results must match
+          both.
         $ref: "#/definitions/ResultFilters"
+      conceptSetName:
```

Comment:
This might make things  too complicated, but did you consider whether to make this a field on `ResultFilters` instead?